src/pages/__devOnly/DesignSystem.tsx

```diff
@@ -166,10 +166,7 @@ const DesignSystem = () => {
                 <Block $marginBottom={theme.spacing(6)}>
                   <Alert
                     fullWidth
-                    containerSize={{
-                      default: 16,
-                      md: 48,
-                    }}
+                    className="md:px-12"
```

Comment:
What about the default size?

Reply:
The default padding in the `Alert` component is already 16px
https://github.com/getlago/lago-front/blob/3e8746197f3bf5704dd1c5fbfcb18a738b465229/src/components/designSystem/Alert.tsx#L38